spring-boot/admin/src/main/java/eu/coatrack/admin/service/OAuthUserDetailsService.java

```diff
@@ -48,26 +48,24 @@
 @RequiredArgsConstructor
 public class OAuthUserDetailsService {
 
-    @Value("${oauth2.attributes.login.name}")
-    private String attributeNameForLogin;
-
-    @Value("${oauth2.attributes.name}")
-    private String attributeNameForName;
-
-    @Value("${oauth2.attributes.company}")
-    private String attributeNameForCompany;
-
-    @Value("${oauth2.attributes.email}")
-    private String attributeNameForEmail;
-
-    private static final String GITHUB_API_EMAILS_URL = "https://api.github.com/user/emails";
     private static final String KEYROCK_COMPANY_ATTRIBUTE = "organizations";
-
     private final OAuth2AuthorizedClientService clientService;
     private final RestTemplate restTemplate;
     private String GITHUB_API_EMAILS_URL;
 
-    @Value("${spring.security.oauth2.client.provider.resource.userInfoUri}")
+    @Value("${ygg.admin.oauth2.attributes.login.username}")
+    private String oauthUserInfoAttributeNameForLogin;
+
+    @Value("${ygg.admin.oauth2.attributes.name}")
+    private String oauthUserInfoAttributeNameForName;
+
+    @Value("${ygg.admin.oauth2.attributes.company}")
+    private String oauthUserInfoAttributeNameForCompany;
+
+    @Value("${ygg.admin.oauth2.attributes.email}")
+    private String oauthUserInfoAttributeNameForEmail;
+
+    @Value("${spring.security.oauth2.client.provider.resource.userInfoUri : null }")
     private String githubApiEndpoint;
```

Comment:
This should not be called `githubApiEndpoint` any more, as it seems generic. Maybe call it `oauthUserInfoUri`?

Reply:
That is a good idea, however please remember that this is only used when CoatRack is deployed with Github

Reply:
Done